### ISSUE TITLE
'ArchwizardModule' does not appear to be an NgModule class.

### ISSUE THREAD

**Issue by Adil1507** [madoar/ng2-wizard#391]
The legacy ViewEngine has been deprecated and removed in Angular 16.

Please add support for Ivy compiler, as your library now breaks with Angular 16 and its blocking us from updating to the newest Angular version 

`'ArchwizardModule' does not appear to be an NgModule class.(-996002)
archwizard.module.d.ts(3, 61): This likely means that the library (angular-archwizard) which declares ArchwizardModule is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so. Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy.`

Versions:
Angular: 16.0.0
angular-archwizard: 7.0.0

**Comment by irg1008** [comment]
I don't think we are getting any update on this library. If you want, or better need it to upgrade, you could  fork it and upgrade it yourself?

**Comment by Adil1507** [comment]
> I don't think we are getting any update on this library. If you want, or better need it to upgrade, you could fork it and upgrade it yourself?

How could we do it? any idea

**Comment by irg1008** [comment]
There is already a fork that fixes the problem. I forked him further to make sure it works for us in our company. You can either fork from him or from me. Or just use his library if you trust him.

If you need it just to test it out my fork is published at [`@nubebytes/angular-archwizard`](https://www.npmjs.com/package/@nubebytes/angular-archwizard)

**Comment by madalinplastoi** [comment]
> There is already a fork that fixes the problem. I forked him further to make sure it works for us in our company. You can either fork from him or from me. Or just use his library if you trust him.
> 
> If you need it just to test it out my fork is published at [`@nubebytes/angular-archwizard`](https://www.npmjs.com/package/@nubebytes/angular-archwizard)

Hi irg1008 and thank you for four published fork.
I am trying to use it (latest version 14.0.2) in an application that just got Angular 17 updated.

Getting the unsupported error
`
 While resolving: client-app@0.0.0
npm ERR! Found: @angular/core@17.1.2
npm ERR! node_modules/@angular/core
npm ERR!   @angular/core@"^17.1.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
**npm ERR! peer @angular/core@">=14 <= 16" from @nubebytes/angular-archwizard@14.0.2**
npm ERR! node_modules/@nubebytes/angular-archwizard
npm ERR!   @nubebytes/angular-archwizard@"^14.0.2" from the root project

`

Is it possible that you release a new version of the package with Angular 17 support?
Best regards,

Madalin
